src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -924,8 +930,10 @@ private void verifyCalculatedData(CompositePurchaseOrder resp) {
       .getCompositePoLines()
       .stream()
       .map(CompositePoLine::getCost)
-      .mapToDouble(Cost::getPoLineEstimatedPrice)
-      .sum();
+      .map(Cost::getPoLineEstimatedPrice)
```

Comment:
Just curious, any reason behind using the same code in unit tests too?
if we are using the exact same calculation that is used in the code for verification too,  will it be fair testing? the value will always end up the same , and isn't it partly the reason why we didn't know about the loss of precision?

Reply:
I agree that having a static expected value to compare against is probably better here.

Reply:
Moved to utility helper.

Reply:
The idea here is to use a assign a value vs the value being calculated even from a utility helper
for ex: double expectedPrice = 49.32 etc..